src/load.js

```diff
@@ -0,0 +1,108 @@
+import ToDo from './todo';
+import {
+  addTodoLs, getTodosls, updateTodosLs, deleteTodoLS, clearLs,
+} from './storage';
+import displayTodos from './display';
+import {
+  edit, updateDescriptionDom, hideEditFormDom, hideDeleteIconDom, hideEllipsyIconDom,
+} from './dom';
+
+const input = document.querySelector('#add-todo-input');
+const form = document.querySelector('#add-todo');
+const toDosArr = getTodosls();
+
+const loadEventListeners = () => {
+  window.addEventListener('DOMContentLoaded', displayTodos(toDosArr));
```

Comment:
:bulb: Here, it's best practice to target the closest element to where the event will occur.  In this case, you should target the document instead of the whole window.